src/containers/Ledgers/index.tsx

```diff
@@ -21,22 +27,45 @@ const LedgersPage = () => {
   const [selected, setSelected] = useState<string | null>(null)
   const [metrics, setMetrics] = useState(undefined)
   const [unlCount, setUnlCount] = useState<number | undefined>(undefined)
-  const { t, i18n } = useTranslation()
+  const [isReady, setIsReady] = useState(false)
+  const { t } = useTranslation()
+  const network = useContext(NetworkContext)
+  const rippledSocket = useContext(SocketContext)
+  const language = useLanguage()
 
   document.title = `${t('xrpl_explorer')} | ${t('ledgers')}`
 
   useEffect(() => {
+    const setIsReadyTrue = () => setIsReady(true)
+    const setIsReadyFalse = () => setIsReady(false)
+    // @ts-ignore
```

Comment:
Why does typescript hate this so much?

Reply:
We should actually use `@ts-expect-error` so that whenever that file is typed, we'll be alerted to remove all the errors.